packages/govuk-frontend/rollup.release.config.mjs

```diff
@@ -30,7 +29,12 @@ export default defineConfig(({ i: input }) => ({
     plugins: [
       terser({
         format: { comments: false },
-        mangle: { reserved: Object.keys(GOVUKFrontend) },
+        mangle: {
+          keep_classnames: true,
+          keep_fnames: true,
+          // Keep our `version` const reserved
+          reserved: ['version']
```

Comment:
Mind if we keep using our export names?

We'll forget to update a hard coded list like this one

```suggestion
          reserved: Object.keys(GOVUKFrontend)
```

Reply:
Fair, I'll put that bit back 😊

Reply:
Brill, will approve when you're ready

Reply:
I'll branch of it to quickly see if fixing the branch checked out solves [our stats problem](https://github.com/alphagov/govuk-frontend/issues/4147). I may hold the merge for a little bit to give us time to investigate (though any of the error branches will bring new code as well, so should show different stats as well).

Reply:
Updated 😊